src/resdk/tables/base.py

```diff
@@ -402,17 +402,31 @@ def _get_orange_data(self) -> pd.DataFrame:
                 # TODO: logging, warning?
                 return pd.DataFrame()
 
-        if "mS#Sample ID" in df.columns:
+        if "Sample ID" in df.columns:
```

Comment:
How about making this even more general? For example,
```
for key in df.columns:
    if "sample id" in key.lower()
        ...
```

Reply:
This PR is merged already. Valid point, i'll add this to one of the future PR's.